js/markdown/shared/MarkdownCode.svelte

```diff
@@ -160,14 +160,4 @@
 	span.md :global(.md-header-anchor > svg) {
 		color: var(--body-text-color-subdued);
 	}
-
-	span :global(h1),
```

Comment:
@pngwn I'm curious why we needed this? It removes any whitespace from the children of headers. If we do need it I could also add margins to those html elements separately.

Reply:
I think it is for the header links + anchors that are generated with certain settings enabled.